### PR TITLE
[firebase_ml_vision] Update ImageDetector for iOS

### PR DESCRIPTION
**Pull request by bparrishMines**
Fixes https://github.com/flutter/flutter/issues/23259 for iOS

Android Companion: https://github.com/flutter/plugins/pull/1367

90% of this is name changing. I included comments explaining name changing of files.

### CODE REVIEW

File: packages/firebase_ml_vision/ios/Classes/ImageLabeler.m [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#import "FirebaseMlVisionPlugin.h"
```
Comment (bparrishMines) [inline packages/firebase_ml_vision/ios/Classes/ImageLabeler.m]: This file was renamed to `ImageLabeler.m` changes can be seen [here](https://github.com/flutter/plugins/pull/1364/commits/c9c42d8175dd30f21243d926165af466e887f590).

File: packages/firebase_ml_vision/lib/src/image_labeler.dart [hunk 2]
```diff
@@ -0,0 +1,127 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```
Comment (bparrishMines) [inline packages/firebase_ml_vision/lib/src/image_labeler.dart]: This file was renamed from `label_detector.dart` changes can be seen [here](https://github.com/flutter/plugins/pull/1364/commits/c9c42d8175dd30f21243d926165af466e887f590).

File: packages/firebase_ml_vision/test/firebase_ml_vision_test.dart [hunk 3]
```diff
@@ -625,175 +621,6 @@ void main() {
       });
     });
 
-    group('$LabelDetector', () {
```
Comment (bparrishMines) [inline packages/firebase_ml_vision/test/firebase_ml_vision_test.dart]: The code below was moved to its own file `image_labeler_test.dart` changes can be seen [here](https://github.com/flutter/plugins/pull/1364/commits/c9c42d8175dd30f21243d926165af466e887f590).

File: packages/firebase_ml_vision/test/image_labeler_test.dart [hunk 4]
```diff
@@ -0,0 +1,165 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```
Comment (bparrishMines) [inline packages/firebase_ml_vision/test/image_labeler_test.dart]: The code below was moved to its own file from `firebase_ml_vision_test.dart` changes can be seen [here](https://github.com/flutter/plugins/pull/1364/commits/c9c42d8175dd30f21243d926165af466e887f590).

File: packages/firebase_ml_vision/example/ios/Podfile [hunk 5]
```diff
@@ -0,0 +1,68 @@
+# Uncomment this line to define a global platform for your project
+platform :ios, '9.0'
```
Comment (bparrishMines) [inline packages/firebase_ml_vision/example/ios/Podfile]: Everything except this line is generated in this `PodFile`.

File: packages/firebase_ml_vision/ios/Classes/ImageLabeler.m (hunk 1 above)
Comment (collinjackson) [reply packages/firebase_ml_vision/ios/Classes/ImageLabeler.m]: I would probably capitalize the L in MLVision

**Comment by bparrishMines** [comment]
@collinjackson Changing the 'L' in the header name would require adding another filename change to this PR. And we would have to change it on the Android side too since code is generated based on the name we provide in the plugin [pubspec.yaml](https://github.com/flutter/plugins/blob/master/packages/firebase_ml_vision/pubspec.yaml#L21). It may not be worth doing in this PR.

**Comment by collinjackson** [comment]
You're right -- the capitalization issue is pre-existing and I agree that it shouldn't block merging this. Thank you.

**Comment by kuprel** [comment]
Great, I was able to get this to install on iOS by pointing pubspec.yaml to a local directory.  Can the dart pub be updated to 0.7.0 too?

**Comment by collinjackson** [comment]
@kuprel 
It looks like version 0.7.0 has been published to pub. https://pub.dartlang.org/packages/firebase_ml_vision

**Comment by kuprel** [comment]
Saw that, thanks!

**Pull request merged**